shared-libs/transitions/test/unit/transitions/update_clinics.js

```diff
@@ -300,20 +300,84 @@ describe('update clinic', () => {
     };
 
     sinon.stub(db.medic, 'query').resolves({ rows: [{ key: '123' }] });
-    sinon.stub(config, 'get').withArgs('forms').returns({ 'someForm': {} });
+    const stubbedConfig = sinon.stub(config, 'get');
+    stubbedConfig.returns([ {
+      form: 'someForm',
+      messages: [
+        {
+          event_type: 'sys.facility_not_found',
+          recipient: 'reporting_unit',
+          translation_key: 'sys.facility_not_found',
+        }
+      ],
+    }]);
+    stubbedConfig.withArgs('forms').returns({ 'someForm': {} });
     sinon.stub(utils, 'translate').returns('translated');
     sinon.stub(utils, 'getLocale').returns('locale');
 
     return transition.onMatch({ doc }).then(changed => {
       assert(changed);
       assert(!doc.contact);
       assert.equal(doc.errors.length, 1);
-      assert.deepEqual(doc.errors[0], { code: 'sys.facility_not_found', message: 'translated' });
-      assert.equal(utils.translate.callCount, 1);
+      assert.deepEqual(doc.errors[0], {
+        code: 'sys.facility_not_found',
+        message: 'translated'
+      });
+      assert.equal(utils.translate.callCount, 2); // called by messages.addMessage and messages.getMessage
       assert.deepEqual(utils.translate.args[0], ['sys.facility_not_found', 'locale']);
     });
   });
 
+  it('should send a message when when form is not public', () => {
+    const doc = {
+      from: '123',
+      type: 'data_record',
+      form: 'someForm'
+    };
+
+    sinon.stub(db.medic, 'query').resolves({ rows: [{ key: '123' }] });
+    sinon.stub(utils, 'translate').returns('facility not found');
+    const stubbedConfig = sinon.stub(config, 'get');
+    stubbedConfig.returns([ {
+      form: 'someForm',
+      messages: [
+        {
+          event_type: 'sys.facility_not_found',
+          recipient: 'reporting_unit',
+          translation_key: 'sys.facility_not_found',
+        }
+      ],
+    }]);
+    stubbedConfig.withArgs('forms').returns({ 'someForm': {} });
+
+    return transition.onMatch({ doc }).then(changed => {
+      assert(changed);
+      assert(!doc.contact);
+      assert.equal(doc.tasks.length, 1);
+      assert.equal(doc.tasks[0].messages[0].to, '123');
+      assert.equal(doc.tasks[0].messages[0].message, 'facility not found');
```

Comment:
Please add some assertions around translate, to make sure that we're translating the configured message, and not the default one.